### PR TITLE
Fish 9008 run pages tck

### PR DESCRIPTION
**Pull request by breakponchito**
Enabling pages TCK execution using maven configuration. Right now on local I have the following results:

image

this results are equivalent from Glassfish:

image

### CODE REVIEW

File: pages-tck/README.MD [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# How to Run
+1. Go to the tck-download/jakarta-pages-tck folder that is on the root of the jakartaee-10-tck-runners project
+   folder and execute `mvn clean install`
+2. Return to the pages-tck folder on the root of the jakartaee-10-tck-runners project folder
+3. Execute the following command indicating the payara.home in case you are using local distribution and the
+   Profile `mvn verify -Dpayara.home=x -Ppayara-server-managed` or `mvn verify -Dpayara.home=x -Ppayara-server-remote`
```
Comment (Pandrex247) [inline pages-tck/README.MD:6]: Technically you don't need to specify payara.home if you're using the payara-server-managed profile
```suggestion
   Profile `mvn verify -Ppayara-server-managed` or `mvn verify -Dpayara.home=x -Ppayara-server-remote`
```

File: pages-tck/pom.xml [hunk 2]
```diff
@@ -31,281 +32,80 @@
...
+        <dependency>
+            <groupId>javatest</groupId>
+            <artifactId>javatest</artifactId>
+            <version>5.0</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>jakarta.tck</groupId>
+            <artifactId>sigtest-maven-plugin</artifactId>
+            <version>${jakarta.tck.sigtest-maven-plugin.version}</version>
```
Comment (Pandrex247) [inline pages-tck/pom.xml:68]: Already defined in top-level pom
```suggestion
```

**Review by Pandrex247 — COMMENTED** [review]
Just a couple of comments

File: pages-tck/pom.xml (hunk 2 above)
Comment (breakponchito) [reply pages-tck/pom.xml]: already done!!

**Comment by breakponchito** [comment]
passing after fixing comments from PR: https://jenkins.payara.fish/job/JakartaEE-11-TCK-Development-FISH-9008/6/

**Review by Pandrex247 — APPROVED** [review]

**Pull request merged**